build/yamls/base/crds.yml

```diff
@@ -155,6 +165,8 @@ spec:
                               type: integer
                             flags:
                               type: integer
+                  not:
+                    required: ["ipHeader", "ipv6Header"]
```

Comment:
what does not.required mean? they can be not set at the same time?

Reply:
Yes, only one can exist, and user will get error when both is set.